src/MemStore.cc

```diff
@@ -212,6 +212,14 @@ MemStore::getStats(StoreInfoStats &stats) const
     stats.mem.size =
         Ipc::Mem::PageLevel(Ipc::Mem::PageId::cachePage) * pageSize;
     stats.mem.count = currentCount();
+    if (Config.workers > 1 && KidIdentifier < Config.workers) {
```

Comment:
Thank you for advancing this PR further. We are making progress, but this condition is incorrect. To save time, I will not go into details, but, for example, it results in unexpected zero "Hot Object Cache Items" stats for `squid -N` instances with the following directives in squid.conf:

```
workers 2
memory_cache_shared on
```

I am working on fixing this. The ball is on my side now, and I expect to push fixes for this (and other/minor) problems today.

Reply:
Done. I pushed and updated PR title/description accordingly. @cvuosalo, please review/adjust and retest.